tests/Spec.hs

```diff
@@ -0,0 +1,15 @@
+module Spec where
+
+import Data.Maybe (fromJust)
+import MoodTracker.Parser (Mood (Bad), MoodEntry (MoodEntry), moodParse, moodWhat, moodWhen, myParseTimeM)
+import Test.Hspec (describe, hspec, it, shouldBe)
+
+main :: IO ()
+main = do
+  file <- readFileLBS "tests/test.jsonl"
```

Comment:
Do you really need to read from an external file, or can this be specified inline in Haskell?

Reply:
You can however it's easier to add more lines to test to an external file. Then it is to write a string inline with multiple escape characters. It makes it easier to read.